Nos/Views/ProfileEdit/CreateUsernameWizard/ExternalNIP05/AlreadyHaveANIP05View.swift

```diff
@@ -0,0 +1,199 @@
+import Combine
+import Dependencies
+import Logger
+import SwiftUI
+
+struct AlreadyHaveANIP05View: View {
+    @Binding var isPresented: Bool
+    @StateObject private var usernameObserver = UsernameObserver()
+    @State private var verified: Bool?
+    @State private var isVerifying = false
+    @Dependency(\.namesAPI) private var namesAPI
+    @Dependency(\.currentUser) private var currentUser
+
+    var body: some View {
+        WizardSheetVStack {
+            Button {
+                isPresented = false
+            } label: {
+                PlainText(.localizable.cancel)
+                    .font(.clarity(.medium, textStyle: .body))
+                    .foregroundStyle(Color.primaryTxt)
+                    .padding(.vertical, 20)
+                    .padding(.leading, -20)
+            }
+            ScrollView(.vertical, showsIndicators: false) {
+                VStack(alignment: .leading, spacing: 20) {
+                    WizardSheetTitleText(.localizable.linkYourNIP05Title)
+                    WizardSheetDescriptionText(markdown: .localizable.linkYourNIP05Description)
+                    HStack {
+                        UsernameTextField(usernameObserver: usernameObserver)
+                            .onChange(of: usernameObserver.debouncedText) { _, newValue in
+                                Task {
+                                    await verify(newValue)
+                                }
+                            }
+                            .onSubmit {
+                                Task {
+                                    await verify(usernameObserver.text)
+                                }
+                            }
+                    }
+                    if linkFailed {
```

Comment:
You could replace this `if ... else` block with `unableToLinkUsernameText().hidden(!linkFailed)`

Reply:
`hidden()` doesn't accept a `bool`

Reply:
oh, haha. Well, it should 😅